lms/djangoapps/teacher_dashboard/views.py

```diff
@@ -65,25 +73,23 @@ def teacher_dahsboard_handler(request, course_id):
     simulation_pk = params.get('simulation')
 
     if req_type == 'licenses':
-        return license_api_call(course)
+        return license_api_call(course, ccx)
```

Comment:
What does it return? Maybe we can put in docsting or in comment what we are expect?